modules/external-api/src/main/java/org/opencastproject/external/common/ApiFormat.java

```diff
@@ -21,5 +21,5 @@
 package org.opencastproject.external.common;
 
 public enum ApiFormat {
-  JSON, XML
+  JSON, XML, UNSPECIFIED
```

Comment:
What is the reasoning behind UNSPECIFIED? It is not a supported format, is it?

Reply:
This is used when clients send "Accept: application/*" which basically means " I accept any format". This would lead to a server error, otherwise. It happens during the unit tests.

Reply:
I removed UNSPECIFIED as well as the unused XML value.